app/navigation/nonsipp/SharesDisposalNavigator.scala

```diff
@@ -48,10 +48,8 @@ object SharesDisposalNavigator extends JourneyNavigator {
       }
 
     case RemoveShareDisposalPage(srn, index, disposalIndex) =>
-      if (userAnswers
-          .map(HowWereSharesDisposedPages(srn))
-          .filter(_._2.nonEmpty)
-          .isEmpty) {
+      if (!userAnswers
+        .map(HowWereSharesDisposedPages(srn)).exists(_._2.nonEmpty)) {
```

Comment:
@ccatosdevelopment Same question here as above

Reply:
we agreed on this